README.md

```diff
@@ -455,42 +508,44 @@ Lua API documentation
 
 [Back to content](#content)
 
-### Subscribe on events
-----------------------
-NOTE
-    Basically, you don't need those functions, so use them with caution.
-    Incorrect uses of following functions might break async I/O loop!
+### Subscribe to events
+-----------------------
+
+  Warning: Use the following functions with caution as 
+  incorrect calls can break asynchronous I/O loops!
 
-    * on_connect
+  Non-mandatory functions:
 
-    * on_disconnect
+  * on_connect
 
-    * on_publish
+  * on_disconnect
 
-    * on_subscribe
+  * on_publish
 
-    * on_unsubscribe
+  * on_subscribe
 
-    You can find Detailed documentation of those functions at file: mqtt/init.lua
+  * on_unsubscribe
+
+  See the detailed documentation of these functions in the [mqqt.init.lua](../mqqt/init.lua) file.
 
 [Back to content](#content)
 
 ## Performance tuning
 ---------------------
 
+TODO: describe me.
+
 [Back to content](#content)
 
 ## Examples
 -----------
 
-  Connecting to the MQTT broker
-    * examples/connect.lua
+  The [examples/connect.lua](../examples/connect.lua) file shows how to connect 
```

Comment:
404. `examples/producer_consumer_queue.lua` below too. I think it is because of `../`.

Reply:
Resolved in #25